src/wincng.c

```diff
@@ -2410,8 +2410,11 @@ _libssh2_wincng_bignum_from_bin(_libssh2_bn *bn, ULONG len,
         if(bignum) {
             bn->bignum = bignum;
             bn->length = length;
+            return 0;
         }
     }
+
+    return -1;
```

Comment:
```suggestion
        }
    }

    return 0;
```
This fixes our CI:
https://ci.appveyor.com/project/libssh2org/libssh2/builds/49553846